clang/include/clang/ExtractAPI/DeclarationFragments.h

```diff
@@ -199,7 +199,8 @@ class DeclarationFragments {
     return *this;
   }
 
-  DeclarationFragments &replace(std::string NewSpelling, unsigned Position) {
```

Comment:
I would prefer to keep the value semantics version and instead move assign the value, that way callers can move the string in and avoid any copies.